public/data/randyh0120.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "Randy Hattab",
+  "type": "personal",
+  "bio": "Software Engineering FGCU 2020, MBA FGCU 2023  | Full-Stack & Mobile Software Engineer",
```

Comment:
I recommend removing this little space here. What do you think?
```suggestion
  "bio": "Software Engineering FGCU 2020, MBA FGCU 2023 | Full-Stack & Mobile Software Engineer",
```

Reply:
Updated and fixed